dll/win32/kernel32/client/console/readwrite.c

```diff
@@ -683,7 +683,7 @@ IntWriteConsole(IN HANDLE hConsoleOutput,
     if (CaptureBuffer) CsrFreeCaptureBuffer(CaptureBuffer);
 
     /* Retrieve the results */
-    if (Success)
+    if (Success && lpNumberOfCharsWritten)
```

Comment:
```suggestion
#if (NTDDI_VERSION >= NTDDI_VISTA)
    if (Success && lpNumberOfCharsWritten)
#else
    if (Success)
#endif
```

Reply:
That completely defeats the purpose, unless we set those flags on this dll.

Reply:
What would be the alternative then? Done in a Vista+ shim?

Reply:
are we going to re-discuss what was discussed in 2018 again?